.circleci/config.yml

```diff
@@ -1,214 +0,0 @@
-          DB_ENCRYPTION_KEY: MMkDMBfYL0Xoz3Xu1ENs3AkdCZdJoks5PNlUBkK7KDc=
-          DJANGO_DEBUG: False
-          DJANGO_HASHID_SALT: test_hash_id
-          DJANGO_SECRET_KEY: test_secret_key
-          DJANGO_SETTINGS_MODULE: config.settings.test
-          SECURE_SSL_REDIRECT: False
-          SFDX_CLIENT_CALLBACK_URL: circle_test
-          SFDX_CLIENT_ID: circle_test
-          SFDX_CLIENT_SECRET: circle_test
-          SFDX_HUB_KEY: circle_test
-      - image: cimg/redis:6.2
-      - image: cimg/postgres:12.9
-        environment:
-          POSTGRES_DB: circle_test
-          POSTGRES_PASSWORD: circle_test
-          POSTGRES_USER: ubuntu
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - run:
-          name: Run Database Migrations
-          command: |
-            . venv/bin/activate
-            python manage.py migrate --no-input
-      - run:
-          name: Test Python
-          command: |
-            . venv/bin/activate
-            pytest --junitxml=pycov/pytest-junit.xml
-      - store_artifacts:
-          path: htmlcov
-          destination: py-reports
-      - store_test_results:
-          path: pycov
-      - persist_to_workspace:
-          root: .
-          paths:
-            - .coverage
-
-  report-test-coverage:
-    working_directory: ~/metecho
-    docker:
-      - image: ghcr.io/oddbird/pyjs:py3.10-node16
-    environment:
-      COVERALLS_PARALLEL: true
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - run:
-          name: Report Test Coverage
-          command: |
-            if [ -n "${COVERALLS_REPO_TOKEN}" ]; then
-              . venv/bin/activate
-              yarn test:js:report-coverage
-              yarn test:py:report-coverage
-              curl -k "https://coveralls.io/webhook?repo_token=${COVERALLS_REPO_TOKEN}" -d "payload[build_num]=${CIRCLE_BUILD_NUM}&payload[status]=done"
-            fi
-      - run:
-          name: Check Test Coverage
-          command: |
-            . venv/bin/activate
-            yarn test:js:check-coverage
-            yarn test:py:check-coverage
-
-  build-docs:
-    working_directory: ~/metecho
-    docker:
-      - image: ghcr.io/oddbird/pyjs:py3.10-node16
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - run:
-          name: Build Documentation
-          command: |
-            . venv/bin/activate
-            cd docs
-            make html
-
-  deploy-storybook:
-    working_directory: ~/metecho
-    docker:
-      - image: ghcr.io/oddbird/pyjs:py3.10-node16
-    steps:
-      - checkout
-      - attach_workspace:
-          at: .
-      - add_ssh_keys:
-          fingerprints:
-            - 'd6:e7:86:ee:3d:de:12:95:d9:59:63:90:55:4f:da:79'
-      - run:
-          name: Deploy Storybook
-          command: |
-            if [ -n "${DEPLOY_STORYBOOK}" ]; then
-              yarn deploy-storybook --ci
-            fi
```

Comment:
It looks like these two checks are not re-created in the new GH action. I'm not sure there's much value to compiling the documentation in CI, but I think we do want to run the `yarn deploy-storybook --ci` command on pushes to the `main` branch.